### ISSUE TITLE
Document types of supported queries

### ISSUE THREAD

**Issue by SeanKilleen** [akkadotnet/Akka.Persistence.SqlServer#247]
**Please describe what you are trying to understand**

I've got an application using Persistence to SQL Server. I am attempting to project events into a more query-friendly SQL Server structure.

**What did these pages not make clear?**

On the [Persistence Query page](https://getakka.net/articles/persistence/persistence-query.html), the docs state: 

> Each read journal must explicitly document which types of queries it supports. Refer to your journal's plugins documentation for details on which queries and semantics it supports.

As far as I can tell, that means I should check this repo. However, this repo does not appear to say anything about the types of queries and semantics it supports, or any documentation of persistent views.

**How can we do it better?**

Usage documentation for this plugin around creating a persistent query or materialized view, perhaps with an example, would be phenomenal. I assume many like me may be encountering these topics for the first time and may require a little more hand-holding in terms of achieving an objective.

**Comment by ismaelhamed** [comment]
Yes. This is the missing doc: https://doc.akka.io/docs/akka/current/persistence-query-leveldb.html

But since all `SqlCommon` plugins are basically a port of the LevelDb plugin in the JVM, maybe this documentation should be added to the Akka.NET repo instead.

**Comment by Aaronontheweb** [comment]
On mobile now but I'll transfer this issue to the central repo. We should definitely cover this in more detail.
